pkg/rsyslog/init.sh

```diff
@@ -1,20 +1,8 @@
 #!/bin/sh
 
-RSYSLOG_WORK_DIR=/persist/rsyslog
-OLD_DIR=/persist/syslog
-# We need to clean up old state it seems. Use $OLD_DIR as the hint
-if [ -d "$OLD_DIR" ]; then
-    echo "Moving old $OLD_DIR and $RSYSLOG_WORK_DIR out of the way"
-    mv "$OLD_DIR" "$OLD_DIR".old
-    mv "$RSYSLOG_WORK_DIR" "$RSYSLOG_WORK_DIR".old
-fi
-if [ ! -d "$RSYSLOG_WORK_DIR" ]; then
-  mkdir -p $RSYSLOG_WORK_DIR
-  chmod 644 $RSYSLOG_WORK_DIR
-fi
-
-IMGP=$(cat /run/eve.id 2>/dev/null)
-IMGP=${IMGP:-IMGX} /usr/sbin/rsyslogd
+./start-rsyslogd.sh
```

Comment:
I feel this is a redundant line in a sense that we don't need a script that starts an rsyslog -- we need one that *restarts* it if rsyslog is not running.

Looking at it this way allows you to push all the logic back into monitor-rsyslog.sh even if you're starting from scratch @gkodali-zededa

Reply:
Makes sense. Will make this change.